### ISSUE TITLE
Finetune with lidar GT

### ISSUE THREAD

**Issue by zhoujiawei3** [facebookresearch/map-anything#91]
I would like to fine-tune the current model on an autonomous driving dataset to obtain more accurate depth estimation. May I directly use the sparse LiDAR image as the depth image to fine-tune the model?

**Comment by zck921031** [comment]
+1

**Comment by lnSong** [comment]
In my experiments, the sparse LiDAR depth data leads to poor performance. Do you have any good solutions?
How is the issue of dynamic objects addressed in the processing?

**Comment by Nik-V9** [comment]
Hi, yes this should be possible and would be akin to how we use some other training datasets like MPSD. Also, we don't deal with dynamic objects in a special way.

My recommendation for training on sparse label data is to mix it with dense label supervision from other synthetic datasets like ParallelDomain4D.
